user_management/src/user_management/settings.py

```diff
@@ -157,7 +148,7 @@
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
 # #SQLite 3 for debugs and tests
-DEBUG = False
+DEBUG = True
```

Comment:
Do not let `DEBUG=True` in production

Reply:
https://github.com/tdameros/42-transcendence/pull/96/commits/deb603477bb8f058ae2f03f3690e7c6ab06f18a5